_layouts/default.html

```diff
@@ -1,47 +0,0 @@
-<!-- SPDX-License-Identifier: CC-BY-4.0 -->
```

Comment:
I noticed these files were out of sync with .gitignore in https://github.com/odpi/egeria/issues/773 - however I wasn't sure of the impact of deletion since they appear to relate to the website. Have you been able to confirm the effect & check with the owner? If it's known good perhaps worth adding a comment to that issue too